### PR TITLE
Boot common scripts rework

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Needs rebase.

**Comment by igoropaniuk** [comment]
@ricardosalveti rebased

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common-alternative.cmd.in [hunk 1]
```diff
@@ -130,6 +130,8 @@ fi
 setenv fiovb.old_is_secondary_boot ${fiovb.is_secondary_boot}
 run check_secondary_boot
 
+if test "${ostree_deploy_usr}" = "1"; then setenv image_dir_path "${bootdir}"; else setenv image_dir_path "${ostree_root}/usr/lib/firmware"; fi;
```
Comment (MrCry0) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common-alternative.cmd.in:133]: Would you mind splitting this commit?

Comment (igoropaniuk) [reply meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common-alternative.cmd.in:133]: done

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by angolini — APPROVED** [review]

File: meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common-alternative.cmd.in (hunk 1 above)
Comment (ricardosalveti) [inline meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common-alternative.cmd.in:133]: We can simplify the logic currently customized at board level:

```
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx8qm-mek/boot.cmd:setenv update_primary_image1 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader_s_image}"; fi; setenv start_blk "${bootloader_s}";  run load_image; run update_image_boot0'
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx8qm-mek/boot.cmd:setenv update_primary_image2 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader2_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader2_s_image}"; fi; setenv start_blk "${bootloader2_s}";  run load_image; run update_image_boot0'
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx8ulp-lpddr4-evk/boot.cmd:setenv update_primary_image1 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader_s_image}"; fi; setenv start_blk "${bootloader_s}";  run load_image; run update_image_boot0'
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/imx8ulp-lpddr4-evk/boot.cmd:setenv update_primary_image2 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader2_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader2_s_image}"; fi; setenv start_blk "${bootloader2_s}";  run load_image; run update_image_boot0'
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx8/boot.cmd:setenv update_primary_image1 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader_s_image}"; fi; setenv start_blk "${bootloader_s}";  run load_image; run update_image_boot0'
meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx8/boot.cmd:setenv update_primary_image2 'if test "${ostree_deploy_usr}" = "1"; then setenv image_path "${bootdir}/${bootloader2_s_image}"; else setenv image_path "${ostree_root}/usr/lib/firmware/${bootloader2_s_image}"; fi; setenv start_blk "${bootloader2_s}";  run load_image; run update_image_boot0'
```

Mind cleaning this up as a consequence of this change (in another patch)?

**Pull request merged**